R/bs-dependencies.R

```diff
@@ -159,6 +98,90 @@ bs_theme_dependencies <- function(
   )
 }
 
+maybe_precompiled_css <- function(theme, sass_options, precompiled) {
+  if (!precompiled) return()
+  if (!identical(sass_options, sass_options(output_style = "compressed"))) {
+    return()
+  }
+
+  precompiled_css <- precompiled_css_path(theme)
+  if (is.null(precompiled_css)) {
+    return()
+  }
+
+  version <- theme_version(theme)
+  deps_hash <- rlang::hash(htmlDependencies(as_sass(theme)))
```

Comment:
This is probably over-thinking things. The one way that the theme could change between calls is if the Bootstrap JavaScript assets change. Taking the hash of the dependencies was intended to catch this, but we'd actually have to hash the files themselves. Not sure if it's worth it.

Reply:
Not hard, maybe still not worth it: 58f62cd46067c58462541ef2bbe37151a7a2cd1f